src/main/java/view/GameView.java

```diff
@@ -0,0 +1,155 @@
+        System.out.println(lottos.size() + PURCHASED);
+    }
+
+    private static int inputMoney() {
+        try {
+            printPleaseInputMoney();
+            String money = sc.nextLine();
+            return Integer.valueOf(money);
+        } catch (NumberFormatException e) {
+            sc = new Scanner(System.in);
+            return inputMoney();
+        }
+    }
+
+    private static void printPleaseInputMoney() {
+        System.out.println(PLEASE_INPUT_MONEY);
+    }
+
+    private static void printLottos(List<Lotto> lottos) {
+        lottos.stream()
+                .map(Lotto::getNumbers)
+                .forEach(System.out::println);
+    }
+
+    public static WinningLotto inputWinningNumbersAndBonusNo() {
+        try {
+            List<Integer> winningNumbers = inputWinningNumbers();
+            int bonusNo = inputBonusNo();
+            return new WinningLotto(new Lotto(winningNumbers), bonusNo);
+        } catch (IllegalArgumentException e) {
+            printPleaseInputAgain();
+            return inputWinningNumbersAndBonusNo();
+        }
+    }
+
+    private static List<Integer> inputWinningNumbers() {
+        try {
+            printPleaseInputWinningNumbers();
+            String input = sc.nextLine();
+            return Arrays.stream(input.split(","))
+                    .mapToInt(Integer::parseInt)
+                    .boxed()
+                    .collect(toList());
+        } catch (NumberFormatException e) {
+            return inputWinningNumbers();
+        }
+    }
+
+    private static void printPleaseInputWinningNumbers() {
+        System.out.println(PLEASE_INPUT_WINNING_NUMBERS);
+    }
+
+    private static int inputBonusNo() {
+        try {
+            printPleaseInputBonusNo();
+            int bonusNo = sc.nextInt();
+            return bonusNo;
+        } catch (InputMismatchException e) {
+            sc = new Scanner(System.in);
+            return inputBonusNo();
+        }
+    }
+
+    private static void printPleaseInputBonusNo() {
+        System.out.println(PLEASE_INPUT_BONUS_BALL);
+    }
+
+    private static void printPleaseInputAgain() {
+        System.out.println(PLEASE_INPUT_AGAIN);
+    }
+
+    public static void printStatistics(WinningLotto winningLotto, Lottos lottos) {
+        System.out.println(WINNING_STATISTICS);
+        System.out.println(PARTIONING_LINE);
+
+        Map<Rank, Long> statistics = lottos.getStatistics(winningLotto);
+        printRankCount(statistics);
+        printYield(lottos.calculateYield(winningLotto));
+    }
+
+    private static void printRankCount(Map<Rank, Long> statistics) {
+        List<String> template = setStatisticsTemplate();
+        statistics.remove(Rank.NONE);
+        IntStream.range(0, template.size())
+                .map(index -> template.size() - 1 - index)
+                .mapToObj(index -> template.get(index) + statistics.getOrDefault(Rank.values()[index], 0L) + COUNTS)
```

Comment:
```suggestion
        Collections.reverse(template);
        IntStream.range(0, template.size())
                .mapToObj(index -> template.get(index) + statistics.getOrDefault(Rank.values()[index], 0L) + COUNTS)
                .forEach(System.out::println);
```

map() 대신에 Collections.reverse() 함수를 사용해서 template를 역순으로 변경하는 것이 더 의미상 명확할 것 같습니다. 
코드를 처음보는 입장에서 느낀점은 map을 통해서 순서를 역순으로 뒤집는 과정을 이해하기 힘들었습니다 🤔.

Reply:
정말 좋은 피드백 감사드립니다. 훨씬 더 가독성이 좋은 코드를 만들 수 있게 되었습니다.